lib/pymedphys/_experimental/pinnacle/pinnacle.py

```diff
@@ -187,7 +187,7 @@ def images(self):
         return self._images
 
     @staticmethod
-    def export_struct(plan, export_path="."):
+    def export_struct(plan, export_path=".", skip_pattern="^$"):
```

Comment:
What if this instead took a list of regex filters, and then on the CLI allow it to be passed multiple times to provide multiple ignore strings?

Reply:
Hi Simon, yeah, that's a possibility... I've been running it with -r  '.*_obj|.*_ring|.*_sm|^sm$|.*_olap|normal_tissue|Skin|RadCalc' but could use it multiple times, with the presumed behaviour of ORing them all together. You'd lose some of the expressive power of the regex, but that's not a problem for my use case. 

Personally... I like big regexes, and I cannot lie :-)

I'll add some tests. Will probably need Phil Chlap's help on doing that... next week.

Reply:
Hi @PhilipYarra 
Thanks a lot for this submission! At first glance this is looking good! As @SimonBiggs mentioned we can have a look at adding a test for this. I'd be happy to help you out with that a bit later in the week.

I agree with Simon that it might be easier for users to allow them to specify -r multiple times since many users might not be familiar with regex's. But I think we should be able to get the best of both worlds by still supporting regex's within the -r strings...

I'll get back to you on Thurs or so with a more detailed code review as well as a start on some unit tests.

Thanks
Phil